pytorch_lightning/callbacks/quantization.py

```diff
@@ -219,3 +264,28 @@ def on_fit_end(self, trainer, pl_module):
             pl_module.forward = wrap_quantize_forward_context(model=pl_module, func=self.__module_forward)
         else:
             pl_module.forward = self.__module_forward
+
+    def on_validation_start(self, trainer, pl_module):
+        self._maybe_disable_observer(pl_module, 'validate' in self._disable_observers and not trainer.sanity_checking)
+
+    def on_validation_end(self, trainer, pl_module):
+        if 'validate' in self._disable_observers:
+            if trainer.sanity_checking:
```

Comment:
If we disactivated them on sanity checking, we don't need this logic right ?

Reply:
Yes, we could remove this if the observers are disabled during the sanity checking.